libparsec/crates/client/src/certif/block_validate.rs

```diff
@@ -146,6 +146,11 @@ pub(super) async fn validate_block(
             CertifForReadWithRequirementsError::InvalidCertificate(err) => {
                 CertifValidateBlockError::InvalidCertificate(err)
             }
+            CertifForReadWithRequirementsError::InvalidRequirements => {
+                CertifValidateBlockError::Internal(anyhow::anyhow!(
```

Comment:
```rust
// This shouldn't occur since `needed_realm_certificate_timestamp` is provided by the server along with the block to validate (and the server is expected to only provide us with valid requirements !).
```

Reply:
beb24ac032ac8203300756322ddacdd2bdef9499